module/VuFindTheme/src/VuFindTheme/View/Helper/HeadScript.php

```diff
@@ -114,6 +114,9 @@ public function itemToString($item, $indent, $escapeStart, $escapeEnd)
         }
 
         $this->addNonce($item);
+        if ($this->view && $this->view->plugin('doctype')->isHtml5()) {
+            $item->type = '';
```

Comment:
Just out of curiosity, what is the purpose of this change? Perhaps a comment would be worthwhile here.

Reply:
It was a misguided attempt at getting rid of the `type="text/javascript"` attribute, which the Laminas helper actually takes care of already. Now removed. Thanks for pointing it out!